site/docs/components/switch/accessibility.mdx

```diff
@@ -0,0 +1,39 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Best practices
+
+- On focus, the screen reader should:
+  - Identify the switch
+  - Read out the text
+  - Express the state
+
+### Keyboard interactions
+
+<KeyboardControls>
+<KeyboardControl keyOrCombos="Tab">
+
+- If focus is on the previous element in the tab order, move focus onto Switch.
+- If focus is on Switch, moves focus to the next focusable element within the tabbing order.
```

Comment:
not sure why here is "within the tabbing order", later is "in the tab order|

Reply:
@sion-docs @mikearildbrown do you have a preference how we address this, or are we ok keeping them as is?